src/main/scala/viper/server/core/ViperCache.scala

```diff
@@ -298,7 +299,9 @@ object ViperCacheHelper {
   protected def hex(h: String): String = h.hashCode.toHexString
 
   /**
-    * This method is used for computing unique-ish hashes of AST nodes.
+    * This method is used for computing unique-ish hashes of AST nodes. `idx` represents
```

Comment:
Can you explain in a bit more detail what "unique-ish" means here? E.g., under which conditions would the hash not be unique?